openstates/tx/votes.py

```diff
@@ -54,35 +54,43 @@ def clean_journal(root):
 
 def names(el):
     text = (el.text or '') + (el.tail or '')
+    split_name_list = text.split(';')
+    if len(split_name_list) < 7:
+        # probably failed to properly split on semi-colons; try commas:
+        split_name_list = text.split(',')
 
     names = []
-    for name in text.split(';'):
+    for name in split_name_list:
         name = name.strip().replace('\r\n', '').replace('  ', ' ')
 
         if not name:
             continue
 
-        if name == 'Gonzalez Toureilles':
-            name = 'Toureilles'
-        elif name == 'Mallory Caraway':
-            name = 'Caraway'
-        elif name == 'Martinez Fischer':
-            name = 'Fischer'
-        elif name == 'Rios Ybarra':
-            name = 'Ybarra'
-
+        name = clean_name_special_cases(name)
```

Comment:
This special handling of names was [introduced in 2010](https://github.com/openstates/openstates/commit/05595720f7a637458f20849f01ffde7472e70780). I wonder if it's still relevant... In any case, I moved it to its own function to keep the main function more streamlined.